components/email/src/init.spec.js

```diff
@@ -740,3 +1232,210 @@ describe("Email: Toggle email of sender/recipient", () => {
     });
   });
 });
+
+describe("Should Render Reply Button And Dispatch Event When Clicked", () => {
+  beforeEach(() => {
+    cy.visit("/components/email/src/cypress.html");
+
+    cy.get("nylas-email").first().as("email");
+
+    cy.get("@email").invoke("attr", "show_reply", "true");
+  });
+
+  it("Should Render Reply Button When Passed A Message", () => {
+    cy.get("@email").invoke("attr", "click_action", "default");
+
+    cy.get("@email").then((elements) => {
+      const component = elements[0];
+      component.message = SINGLE_SENDER_MESSAGE;
+      cy.get("@email").should("exist");
+      cy.get("@email").find("div.reply button").should("exist");
+    });
+  });
+
+  it("Should Render Reply Button When Passed A Thread", () => {
+    cy.get("@email").invoke("attr", "click_action", "default");
+    cy.get("@email").invoke("attr", "show_expanded_email_view_onload", "true");
+
+    cy.get("@email").then((elements) => {
+      const component = elements[0];
+      component.thread = SAMPLE_THREAD;
+      cy.get(component).should("exist");
+      cy.get(component).find("div.reply button").should("exist");
+    });
+  });
+
+  it("Should Render Reply Button When Passed A Message ID", () => {
+    cy.intercept(
+      {
+        method: "GET",
+        url: "**/messages/d0byfc378l2728z35pax362ho",
+      },
+      {
+        fixture: "email/messages/id.json",
+      },
+    ).as("messageRequest");
+
+    cy.get("@email").invoke("attr", "message_id", "d0byfc378l2728z35pax362ho");
+    cy.wait("@messageRequest");
+
+    cy.get("@email").should("exist");
+    cy.get("@email").find("div.reply button").should("exist");
+  });
+
+  it("Should Render Reply Button When Passed A Thread ID", () => {
+    cy.intercept(
+      {
+        method: "GET",
+        url: "**/threads/e2k5xktxejdok7d8x28ljf44d?view=expanded",
```

Comment:
This can be a [glob](https://docs.cypress.io/api/commands/intercept#Examples) 
```suggestion
        url: "/threads/e2k5xktxejdok7d8x28ljf44d?view=expanded",
```

Reply:
** is a glob and is necessary to properly to detect url